github/repos.go

```diff
@@ -1321,3 +1321,31 @@ func (s *RepositoriesService) Transfer(ctx context.Context, owner, repo string,
 
 	return r, resp, nil
 }
+
+// DispatchRequest represents a request to trigger a repository_dispatch event.
+type DispatchRequest struct {
+	EventType string  `json:"event_type"`
+}
+
+// Dispatch triggers a repository_dispatch event in a GitHub Actions workflow.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/#create-a-repository-dispatch-event
+func (s *RepositoriesService) Dispatch(ctx context.Context, owner, repo string, dispatch DispatchRequest) (*Repository, *Response, error) {
```

Comment:
Let's please change the name of `dispatch` to `opts` for more consistency with the repo.

Reply:
Hi @fleskesvor, it looks like this comment was not addressed. Could you please fix this?

Reply:
Oh, right. I actually did that before but changed it to dispatch in my last commit, after looking at the function directly above. I will change it back to `opts` again.